src/sympc/tensor/mpc_tensor.py

```diff
@@ -250,6 +250,14 @@ def sub(self, y: Union["MPCTensor", torch.Tensor, float, int]) -> "MPCTensor":
         """
         return self.__apply_op(y, "sub")
 
+    def rsub(self, y: Union["MPCTensor", torch.Tensor, float, int]) -> "MPCTensor":
```

Comment:
NITPICK: Why is ```MPCTensor``` as a possible type? (if ```y``` was an MPCTensor we would call the ```sub``` method)

Reply:
nice pick  :D